### PR TITLE
Stop pending page

### PR DESCRIPTION
**Pull request by kildre**
Implemented the new design for the stop pending page
## Reference Issues or PRs
https://gitlab.jatic.net/jatic/team-metrostar/t-e-platform/-/issues/339

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [X] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [X] Did you test the pull request locally?
- [X] Did you add new tests?

#### Text styling

- [x] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [x] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [x] This content adheres to the Nebari style guides.

#### Non-text content

- [ x] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

## Any other comments?
image

### CODE REVIEW

File: ui/src/pages/stop-pending/stop-pending.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  border: 4px solid rgba(0, 0, 0, 0.1);
+  border-top: 4px solid #a020f0; /* Purple color for the spinner */
+  border-radius: 50%;
+  animation: spin 1s linear infinite;
+}
+
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+}
```
Comment (jbouder) [inline ui/src/pages/stop-pending/stop-pending.css:25]: Why is this needed? Doesn't the circular progress work on its own?

File: ui/src/pages/stop-pending/stop-pending.test.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    ).toBeInTheDocument();
+    expect(
+      screen.getByRole('button', { name: /Back To Home/i }),
+    ).toBeInTheDocument();
+  });
+
+  test('matches snapshot', () => {
+    const { asFragment } = render(componentWrapper);
+    expect(asFragment()).toMatchSnapshot();
+  });
```
Comment (jbouder) [inline ui/src/pages/stop-pending/stop-pending.test.tsx:51]: I don't think we should be using snapshots. Unless you have a strong feeling, lets remove this and the snapshot

File: ui/src/pages/stop-pending/stop-pending.css (hunk 1 above)
Comment (kildre) [reply ui/src/pages/stop-pending/stop-pending.css:25]: That was a spinner before Mui spinner, removed

File: ui/src/pages/stop-pending/stop-pending.test.tsx (hunk 2 above)
Comment (kildre) [reply ui/src/pages/stop-pending/stop-pending.test.tsx:51]: Removed snapshot

File: ui/src/pages/stop-pending/stop-pending.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
+Copy code
```
Comment (jbouder) [inline ui/src/pages/stop-pending/stop-pending.css:1]: What is this?

Comment (kildre) [reply ui/src/pages/stop-pending/stop-pending.css:1]: I don't know what that is, removed.

**Review by jbouder — APPROVED** [review]
